openjdk/excludes/ProblemList_openjdk17-openj9.txt

```diff
@@ -225,7 +225,7 @@ java/nio/Buffer/DirectBufferAllocTest.java https://github.com/eclipse-openj9/ope
 java/nio/Buffer/LimitDirectMemory.java	https://github.com/eclipse-openj9/openj9/issues/8063	generic-all
 java/nio/Buffer/LimitDirectMemoryNegativeTest.java	https://github.com/eclipse-openj9/openj9/issues/8065	generic-all
 java/nio/channels/DatagramChannel/AdaptorMulticasting.java https://github.ibm.com/runtimes/backlog/issues/655 windows-all,linux-s390x,macosx-all
-java/nio/channels/AsyncCloseAndInterrupt.java	https://github.com/adoptium/infrastructure/issues/1173	linux-all,aix-all
+java/nio/channels/AsyncCloseAndInterrupt.java	https://github.com/adoptium/infrastructure/issues/1173	linux-all,aix-all,z/OS-s390x
```

Comment:
This is permanently excluded on zOS, right? It is not related to 1173. Could you add a comment for this? Thanks

Reply:
@llxia Added the comment as you suggested. Thank you.